test/catch.hpp

```diff
@@ -0,0 +1,12012 @@
+/*
+ *  Catch v2.0.1
```

Comment:
To jest framework do testów jednostkowych. Po prostu najprościej jest go wrzucić do projektu zamiast męczyć się z cmake'iem żeby go zawsze pobierał i trzymał jako shared czy coś.